### PR TITLE
casperjs 1.1.2

### PR DESCRIPTION
**Pull request by nijikon**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/casperjs.rb [hunk 1]
```diff
@@ -23,36 +14,16 @@ class Casperjs < Formula
...
+  depends_on "phantomjs"
 
   def install
     libexec.install Dir["*"]
-    (libexec/"phantomjs").install resource("phantomjs")
 
     (bin/"casperjs").write <<-EOS.undent
       #!/bin/bash
-      export PATH=#{libexec}/phantomjs/bin:$PATH
+      export PATH=#{Formula["phantomjs"].opt_bin}:$PATH
```
Comment (DomT4) [inline Formula/casperjs.rb]: If it's not using a custom `phantomjs` now & will actually work against our `phantomjs` okay you can just remove the script & go back to a more simple symlink mechanism.

File: Formula/casperjs.rb [hunk 2]
```diff
@@ -23,38 +14,12 @@ class Casperjs < Formula
...
-
   # For embedded Phantomjs
   depends_on :macos => :snow_leopard
+  depends_on "phantomjs"
 
   def install
     libexec.install Dir["*"]
-    (libexec/"phantomjs").install resource("phantomjs")
-
-    (bin/"casperjs").write <<-EOS.undent
```
Comment (dunn) [inline Formula/casperjs.rb]: Since you removed this there's nothing in `bin` which is why the test failed.

**Comment by nijikon** [comment]
@DomT4 @dunn please review

**Comment by nijikon** [comment]
@dunn please review

**Comment by DomT4** [comment]
Seems alright. It wouldn't surprise me if we end up resourcing `phantomjs` again at some point in future, but this seems to be passing the test at least.

**Comment by nijikon** [comment]
👍

**Pull request closed without merging**